18F/identity-idp
# LG-7483: Remove feature flag select_multiple_mfa_options

## 🎫 Ticket

[LG-7483](https://cm-jira.usa.gov/browse/LG-7483)

## 🛠 Summary of changes

Removes the `select_multiple_mfa_options` feature flag, which is now live in all environments.

Also fixes an issue where MFA "Delete" links would not be shown on the account dashboard in some circumstances, since that was intended to apply only for Kantara restrictions (not live in production), but instead had been keying off the `select_multiple_mfa_options` feature (live in production).

## 📜 Testing Plan

1. Check that you can configure multiple MFA for account creation
2. Check that you can delete MFA methods from your account when you have multiple active, regardless of whether the MFA method is "restricted"

The second test procedure requires overrides in local development `config/application.yml` (until LG-7586):

```
development:
  kantara_2fa_phone_restricted: false
```
